### ISSUE TITLE
Uncaught TypeError: Cannot read property 'className' of null

### ISSUE THREAD

**Issue by Metalmi** [nhn/tui.image-editor#411]
**Describe the bug**
Console error when clicking between text style buttons:
```
Uncaught TypeError: Cannot read property 'className' of null
    at Text._setTextEffectHandler (text.js:163)
```
![image](https://user-images.githubusercontent.com/35292893/84588900-751ec900-ae2b-11ea-80b7-8ef3481934d4.png)

**To Reproduce**
Steps to reproduce the behaviour:
1. Go to [Toast UI Image Editor](https://ui.toast.com/tui-image-editor/)
2. Click on "Text" menu item
3. Click between "Bold" and "Italic", or between "Italic" and "Underline"

**Expected behaviour**
No error happens.

**Desktop:**
 - OS: Windows 10
 - Browser: all
 - Version: 3.9.0

**Comment by jinwoo-kim-nhn** [comment]
Thanks. I will correct it after checking.

**Comment by lja1018** [comment]
@Metalmi 
Thanks for the report. Resolved in [v3.12.0](https://github.com/nhn/tui.image-editor/releases/tag/v3.12.0).
